qemu/tests/uefishell.py

```diff
@@ -76,6 +80,24 @@ def copy_uefishell(self):
             process.system(cp_command)
         return uefishell_dst_path
 
+    def copy_secureboot_pk_kek(self):
+        """
+        Copy SecureBootPkKek1.oemstr
+        :return SecureBootPkKek1.oemstr path
+        """
+        pk_kek_filepath = data_dir.get_deps_dir("edk2")
+        pk_kek_filename = "SecureBootPkKek1.oemstr"
```

Comment:
If there will be more than one 'secureBootPkKet1.oemstr' file, I suggest that add the filename into the cfg instead of hard code here

Reply:
agree with you, I will update it. thanks